docs/src/getting-started/basics.md

```diff
@@ -276,6 +276,35 @@ let contract_id = "0x0123..." // Your contract ID as a string.
 let connected_contract_instance = MyContract::new(contract_id, wallet);
 ```
 
+## Getting the contract call outputs
+
+### Contract call success
+
+If the contract call succeeded, getting the receipts and logs is done this way:
+
+```rust,ignore
+let result = contract_instance.my_method(args).call().await?;
+logs = result.logs.unwrap() // This gives out the decoded hex LOGD logs
+receipts = result.receipts // This gives out all the receipts of the transaction
```

Comment:
@Br1ght0ne moved examples into the `examples/` directory with #202. In order to keep these docs up to date, all new code examples, even one-liners, should be placed there.

Reply:
In the docs, some examples are not from the `examples` directory. But maybe everything should be you're right.

Reply:
Ah I guess it can be deferred to a future PR with #342